hybrid/go.mod

```diff
@@ -0,0 +1,129 @@
+module github.com/hewlettpackard/hybrid
+
+go 1.19
+
+replace github.com/spiffe/spire => github.com/spiffe/spire v1.2.1-0.20221027032717-3cbfb6277d23
```

Comment:
I think you can add the SPIRE dependency without having to use replace.

Reply:
When they release the new version, i'll remove this replace. It was done like this because some dependency was removed.